assembl/static2/css/components/media.scss

```diff
@@ -0,0 +1,141 @@
+.media-section {
+  padding-bottom: $margin-section-l;
+  padding-top: $margin-section-m;
+
+  .media-title-section {
+    margin-left: 6.5%;
+    margin-right: 6.5%;
+    padding-bottom: $margin-section-ml;
+
+    .dark-title-1 {
+      padding-top: 9px;
+      font-family: $font-family4;
+      font-size: $font-size-xl;
+      font-weight: 300;
+      letter-spacing: 2px;
+      line-height: 56px;
+    }
+  }
+
+  .container-fluid {
+    margin-left: 12.5%;
+    margin-right: 15.5%;
+
+    .media-description-layer {
+      text-align: justify;
+      font-family: $font-family4;
+      font-size: $font-size-m;
+      line-height: 32px;
+      letter-spacing: 1px;
+      font-weight: 300;
```

Comment:
You have a dedicated font-familiy to handle the bold. You shouldn't use font-weight in the project, but maybe there is a good reason

Reply:
No good reason you're right, it's already the font-weight of $font-family4 aka LatoLight

Reply:
It's artifacts from copy pasting the invision css